Consul/Properties/AssemblyInfo.cs

```diff
@@ -1,18 +1,5 @@
-using System.Reflection;
 using System.Runtime.CompilerServices;
 
-// General Information about an assembly is controlled through the following
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyTitle("Consul.NET")]
-[assembly: AssemblyDescription(".NET API for Consul (http://www.consul.io/)")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("G-Research Limited")]
```

Comment:
This will be taken from the `Authors` property so will become `"PlayFab,G-Research"` for Consul. I guess that's fine?

Reply:
I guess so